packages/core/src/scss/components/checkout/paymentProvider/_paymentProvider.scss

```diff
@@ -44,6 +44,14 @@
     flex-shrink: 1;
 }
 
+@media (max-width: 480px) {
```

Comment:
Can we try and use one of the pre-defined variables here https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/scss/settings/global/screensizes/_screensizes.scss

Reply:
I changed it to predefined variable